Rick&Morty/RickAndMorty/Scenes/Episodes/EpisodeDetail/EpisodeDetailView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  EpisodeDetailView.swift
+//  RickAndMorty
+//
+//  Created by Jan Gutwirth on 02.09.2022.
+//  Copyright © 2022 STRV. All rights reserved.
+//
+
+import SwiftUI
+
+struct EpisodeDetailView: View {
+    let episode: Episode
+    let mockedCharacters = Character.characters
+    
+    enum Event {
+        case didTapToOpenInSafari(episode: Episode)
+    }
+    
+    weak var coordinator: EpisodeDetailEventHandling?
+    
+    var formattedAirDate: String {
+        episode.airDate.formatted(
+            .dateTime
+                .month(.wide)
+                .day(.defaultDigits)
+                .year()
+        )
+    }
+    
+    var body: some View {
+        ZStack {
+            BackgroundGradientView()
+            
+            content
+        }
+        .navigationTitle(episode.name)
+        .foregroundColor(.appTextBody)
+        .preferredColorScheme(.none)
+        .toolbar {
+            ToolbarItem {
+//                Button(action: toggleMode) {
+//                    mode.image
+//                }
+            }
+        }
+    }
+    
+    var content: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 8) {
+                Text(R.string.localizable.episodeDetailInfo())
+                    .font(.appSectionTitle)
+                    .foregroundColor(.appTextSectionTitle)
+                
+                info
+                
+                Text(R.string.localizable.episodeDetailAppearingCharacters())
+                    .font(.appSectionTitle)
+                    .foregroundColor(.appTextSectionTitle)
+                
+                appearingCharacters
+            }
+            .padding(.horizontal, 16)
+        }
+    }
+    
+    var info: some View {
+        VStack(alignment: .horizontalInfoAlignment, spacing: 8) {
+            HStack(alignment: .center, spacing: 8) {
+                Image.systemInfoCircle
+                
+                Text(episode.name)
+                    .font(.appItemDescription)
+                    .alignmentGuide(.horizontalInfoAlignment) { $0[.leading] }
```

Comment:
You don't need this

Reply:
But if I remove those three, it doesn't align itself properly 🤔
![Snímek obrazovky 2022-09-09 v 12 04 52](https://user-images.githubusercontent.com/72510054/189326735-1a08ecaf-74c5-42e1-968c-56d89dae22bb.png)

Reply:
You are right 🙂